src/pages/cookbooks/[cookbookId]/recipes/[recipeId]/sections/[sectionId].tsx

```diff
@@ -1,11 +1,45 @@
 import { useRouter } from 'next/router';
-import { FunctionComponent } from 'react';
+import { useEffect } from 'react';
+import { useStore } from '../../../../../../store/store';
+import { CookbookSection } from '../../../../../../components/Cookbook/CookbookGuide/CookbookSection/CookbookSection';
+import { SectionModel } from '../../../../../../models/Section';
 
-export const Section: FunctionComponent = () => {
+const init = async (sectionId: string | string[]) => {
+  if (sectionId != null) {
+    // TODO: Should probably get the specific section from a section array, not the guides
+
+    useStore.getState().guides.forEach(guide => {
```

Comment:
I'm unexperienced with zustand but you could look into the `selector` concept.
https://github.com/pmndrs/zustand#using-subscribe-with-selector

Basically making a method on your state that retreives something, for example the section based on its id.
I'm unsure if this applies here but at least it does in Redux
https://redux.js.org/usage/deriving-data-selectors#basic-selector-concepts